### ISSUE TITLE
SMTP Amazon Simple Email Service (SES)

### ISSUE THREAD

**Issue by geosmarty** [geosmarty/geocore-community#18]
[vinprom1](https://github.com/vinprom1) commented [Apr 8, 2022](https://github.com/geodesicsolutions-community/geocore-community/issues/161#issue-1197202226)

Hello, i want send emails with Amazon Simple Email Service (SES) SMTP
but Main Email Sender addon not send it.
With mailjet.com or other providers SMTP work fine, only Amazon SES not work.

**Comment by geosmarty** [comment]
[vinprom1](https://github.com/vinprom1) commented [Apr 15, 2022](https://github.com/geodesicsolutions-community/geocore-community/issues/161#issuecomment-1100057477)

Hi vicos59,
can you update the swiftmailer to v6.0?
Amazon's SES smtp server is very important to me. Without it, my site become almost unusable ;(

**Comment by geosmarty** [comment]
[vicos59](https://github.com/vicos59) commented [Apr 16, 2022](https://github.com/geodesicsolutions-community/geocore-community/issues/161#issuecomment-1100171909) •

I hate to say it, but your best bet may be to hire a coder to fix it for you. They could clone the repository, make the fix, then if you want to contribute back, issue a pull request to have the new code incorporated back into the main repo here.

Just a note:

Swiftmailer is not maintained anymore. [Use Symfony Mailer](https://symfony.com/doc/current/mailer.html) instead. Read more on [Symfony's blog](https://symfony.com/blog/the-end-of-swiftmailer).

https://swiftmailer.symfony.com/docs/introduction.html

The swiftMailer repo:

https://github.com/swiftmailer/swiftmailer

**Comment by geosmarty** [comment]
[vinprom1](https://github.com/vinprom1) commented [Apr 17, 2022](https://github.com/geodesicsolutions-community/geocore-community/issues/161#issuecomment-1100666595)

I try with latest swiftmailer but it still not work with Amazon SES.
I will read more on the Internet about this problem. I think it's a small bug to fix, but ...

**Comment by geosmarty** [comment]
This is an abbreviated version of the thread.  The problem arises where the legacy geodesic software uses a now deprecated swiftmailer wrapper.  Upgrading to version 6 is not advisable given development and maintenance for Swift Mailer ceased November 2021.  Version 6 upgrade required some compatibility changes, however a glaring deficit was the removal of the native mail transport wrapper.  

Instead, upgrade and conversion should be made to use the latest version Symfony Mail - compatible to php8, or consider another mailer such as phpmailer.  You can, instead of using your own SMTP server or sendmail binary, can send emails via a third-party provide such as Amazon SES.

**Comment by vinprom1** [comment]
Yes, I agree with you! But, I'm not that good at programming, if someone who knows can do it that would be great. Of course, for free work, enthusiasts are rare, but for this nice script, I think that such people will be found.

**Comment by 1kiwi1** [comment]
> Yes, I agree with you! But, I'm not that good at programming, if someone who knows can do it that would be great. Of course, for free work, enthusiasts are rare, but for this nice script, I think that such people will be found.

Hi vinprom1.  You might find something free and decide to wait it out until it arrives.  Alternatively, we have built a commercially available script of the original Geocoremax software left behind by GeodesicSolutions when they exited.  There are many fixes and enhancements in there, including this issue in our geocoremax script.  Essentially the old swiftmailer library was replaced with the latest version of Symfony Mailer.  You will note there is facilitation in Symfony for 3rd Party transports to use Amazon SES.  You can check out our demo at **demo.geocoremax.com** Things are still under development **Geocoremax.com**, yet the forum is available at **forum.geocoremax.com**, the demo at **demo.geocoremax.com** and the sales area at **geocoremax.com**.  Keep an eye out for things you need already in the @Geocoremax (GeodesicSolutions) script.

**Comment by vinprom1** [comment]
> Hi vinprom1. You might find something free and decide to wait it out until it arrives. Alternatively, we have built a commercially available script of the original Geocoremax software left behind by GeodesicSolutions when they exited. There are many fixes and enhancements in there, including this issue in our geocoremax script. Essentially the old swiftmailer library was replaced with the latest version of Symfony Mailer. You will note there is facilitation in Symfony for 3rd Party transports to use Amazon SES. You can check out our demo at **demo.geocoremax.com** Things are still under development **Geocoremax.com**, yet the forum is available at **forum.geocoremax.com**, the demo at **demo.geocoremax.com** and the sales area at **geocoremax.com**. Keep an eye out for things you need already in the @Geocoremax (GeodesicSolutions) script.

Thanks! I can take advantage. but it all depends on the price of the script in geocoremax.com
Free support for the geocore-community script on github will continue right?

**Comment by 1kiwi1** [comment]
> > Hi vinprom1. You might find something free and decide to wait it out until it arrives. Alternatively, we have built a commercially available script of the original Geocoremax software left behind by GeodesicSolutions when they exited. There are many fixes and enhancements in there, including this issue in our geocoremax script. Essentially the old swiftmailer library was replaced with the latest version of Symfony Mailer. You will note there is facilitation in Symfony for 3rd Party transports to use Amazon SES. You can check out our demo at **demo.geocoremax.com** Things are still under development **Geocoremax.com**, yet the forum is available at **forum.geocoremax.com**, the demo at **demo.geocoremax.com** and the sales area at **geocoremax.com**. Keep an eye out for things you need already in the @Geocoremax (GeodesicSolutions) script.
> 
> Thanks! I can take advantage. but it all depends on the price of the script in geocoremax.com Free support for the geocore-community script on github will continue right?

You get what you get with open source, no guarantees.
